src/shared/components/Footer/FooterList/component.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { FooterListWrapper, FooterListHeading, SocialIconWrapper, FooterListItem } from './styles';
+import { TwitterIcon, LinkedinIcon, FacebookIcon } from '../../../icons';
```

Comment:
Pls, don't write like  that: `../../../icons`. You can use absolute paths: 
`import { TwitterIcon, LinkedinIcon, FacebookIcon } from 'shared/icons'`